yktrace/src/tir.rs

```diff
@@ -554,4 +616,18 @@ mod tests {
         assert_eq!(res, 15);
         assert!(tir_trace.len() > 0);
     }
+
+    #[test]
+    #[should_panic]
+    fn use_undefined_var() {
+        let outside_var = 100;
+
+        #[cfg(tracermode = "sw")]
+        let tracer = start_tracing(Some(TracingKind::SoftwareTracing));
```

Comment:
Why are we invoking the software tracer here?

Reply:
It would only call the software tracer if we pass `-C tracer=sw` otherwise the line below is used instead.

It's an idiom I copied from a nearby test.

Reply:
I thought it was unnecessary since software tracing is currently dead anyway. But we can leave it in if you like.